### PR TITLE
Error check

### PR DESCRIPTION
**Pull request by lziq**
Issue #37 , if available:

Description of changes:

### CODE REVIEW

File: client/src/test/diagnostics.test.ts [hunk 1]
```diff
@@ -53,6 +53,176 @@ suite('Should get diagnostics', () => {
 			{ message: 'no viable alternative at input \'2004-12-11T12\'', range: toRange(0, 11, 0, 13), severity: vscode.DiagnosticSeverity.Warning, source: 'ex' }
 		]);
 	});
+	test('blob check 1', async () => {
```
Comment (therapon) [inline client/src/test/diagnostics.test.ts:56]: Try to provide a description for the test that explains what is the use case. As it is not I have to go find the file `blob1.ion` and read its contents. 

Perhaps a better name would be `'blob invalid blob contents'` to indicate that the error is due to the blob having an incorrect value vs not closing curly braces correctly

File: client/test/diagnostics/annotation_1.ion [hunk 2]
```diff
@@ -0,0 +1 @@
+"annotation"::value
```
Comment (therapon) [inline client/test/diagnostics/annotation_1.ion:1]: why is this invalid?

File: client/test/diagnostics/annotation_3.ion [hunk 3]
```diff
@@ -0,0 +1 @@
+false::23
```
Comment (therapon) [inline client/test/diagnostics/annotation_3.ion:1]: Why is this invalid?

File: client/test/diagnostics/annotation_1.ion (hunk 2 above)
Comment (lziq) [reply client/test/diagnostics/annotation_1.ion:1]: I think annotation must be a symbol, right?

Comment (therapon) [reply client/test/diagnostics/annotation_1.ion:1]: Yes. Sorry not enough coffee :)

File: client/src/test/diagnostics.test.ts (hunk 1 above)
Comment (lziq) [reply client/src/test/diagnostics.test.ts:56]: Sure, I will correct it later.

File: client/test/diagnostics/annotation_3.ion (hunk 3 above)
Comment (lziq) [reply client/test/diagnostics/annotation_3.ion:1]: I am wondering as well. However, It is in the bad data samples. The link is here: [https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion]. (https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion)

Comment (lziq) [reply client/test/diagnostics/annotation_3.ion:1]: I am wondering as well. However, It is in the bad data samples. The link is here: https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/annotationFalse.ion

**Pull request merged**